getting-started.md

```diff
@@ -1,38 +1,102 @@
+high performance even while scaling storage to normally prohibitive amounts of data. 
+Hypertables can engage in normal table operations, including JOINs with standard 
+tables.
 
-Getting started with TimescaleDB involves a few steps:
+If you know PostgreSQL, you are 90% of the way to knowing TimescaleDB. If 
+you want to learn more, here are some additional resources:
 
-1. [Installing PostgreSQL and TimescaleDB.][install]
-1. [Creating a standard PostgreSQL database and enabling the TimescaleDB extension on the database.][setup]
-1. [Creating a hypertable to store your data.][create-hypertables]
-1. [Migrating your data to a hypertable][migrate] (optional)
-1. [Setting up data node authentication for distributed hypertables][distdb-auth]
+- [Learn more about time-series data][time-series-data] and how you can best use it for your applications.
+- Learn more about [the TimescaleDB architecture][timescaledb-architecture].
+- Learn more about [the unique features of TimescaleDB][using-timescaledb]. 
 
+### Options for installing TimescaleDB
+
+The best way to get TimescaleDB is through our hosted offering. You can 
+[try TimecaleDB for free][try-for-free] and get started in seconds. Hosted 
+TimescaleDB lets you focus on your workloads while we handle the operations 
+and management of your critical time-series data. TimescaleDB is available in 
+the three top cloud providers (Amazon Web Services, Microsoft Azure, and 
+Google Cloud Platform) across 75+ regions and over 2000 different configurations.
+
+You can also [install TimescaleDB][install-timescale] on your desktop or 
+self-managed in your own infrastructure for free.
+
+### Getting familiar with TimescaleDB
+
+There are a lot of things TimescaleDB can do for you and your time-series data. 
+Here are some of our favorite features, along with links to learn more:
+
+- [Migrating your data to a hypertable][migrate] (optional)
+- Analyze your data using advanced [time-series analytical functions][time-series-functions] (e.g., gap filling, LOCF, interpolation)
+- [Native compression][using-compression] can reduce storage by up to 90%, saving you a significant amount of money on your time-series deployment
+- [Continuous aggregates][using-continuous-aggregates] automatically calculate the results of a query in the background and materialize the results
+- [Data retention][using-data-retention] policies help you decide when to store old data based on its relevance to your use case
+- Achieve petabyte scale with [Multi-node][multinode] (distributed hypertables)
+- Support for [high cardinality][high-cardinality] datasets
+- Support for all PostgreSQL extensions, such as [PostGIS][hello-timescale]
+- Compatibility with [Grafana][grafana-tutorials], [Tableau][tableau-tutorials], and most visualization tools
+- Support for [VPC Peering][vpc-peering] (in Timescale Cloud)
+- [SSL Support for database connections][ssl-support] and better security
```

Comment:
So, it seems initially, that this will not be true for multi-node on Forge. don't think that changes saying this, but just wanted to note it.